weldx/measurement.py

```diff
@@ -1,34 +1,25 @@
 """Contains measurement related classes and functions."""
 
 from dataclasses import dataclass, field
-from typing import TYPE_CHECKING, Dict, List, Tuple, Union  # noqa: F401
+from typing import TYPE_CHECKING, Dict, List, Tuple  # noqa: F401
+from warnings import warn
 
-import xarray as xr
+from weldx.asdf.tags.weldx.core.graph import build_graph, build_tree
```

Comment:
so is it not feasible to use the default graph serialization?

Reply:
Should be doable. We can change that.

Reply:
It's now implemented using the default graph serialization